jupyter_firefly_extensions/__init__.py

```diff
@@ -36,13 +37,42 @@ def initialize(self, notebook_dir, firefly_url):
         self.firefly_url = firefly_url
 
     def get(self):
+        log = logging.getLogger(__name__)
         path = self.get_argument('path', 'not found')
-        print('sendToFirefly:uploading: ' + path)
         fc = FireflyClient(url=self.firefly_url)
-        upload_name = fc.upload_file(self.notebook_dir+'/'+path)
-        print('sendToFirefly:done: ')
+        first_try_file = self.notebook_dir+'/'+path
+        second_try_file = os.path.abspath(path)
+        third_try_file = ''
+        if 'HOME' in os.environ:
+            third_try_file = os.environ['HOME'] + '/' + path
+
+        if self.can_read(first_try_file):
+            log.info('sendToFirefly: uploading: ' + first_try_file)
+            upload_name = fc.upload_file(first_try_file)
+        elif self.can_read(second_try):
```

Comment:
`second_try_file`, no?

Reply:
oh, good point...hard to test this code if `first_try_file` works in the test environment.

Reply:
I need to rewrite this part, clean it up